packages/core/tsconfig.json

```diff
@@ -18,5 +18,5 @@
         "baseUrl": "./",
         "experimentalDecorators": true
     },
-    "include": ["src", "global.d.ts", "../../utils.d.ts"]
+    "include": ["src", "../../global.d.ts", "../../utils.d.ts"]
```

Comment:
@hamid-deriv Should do the same thing for the cashier `tsconfig` file as well.

Reply:
Added.